### PR TITLE
Only one carriage return

### PR DESCRIPTION
**Pull request by cclauss**
### **Describe your change:**
* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: data_structures/binary_tree/avl_tree.py [hunk 1]
```diff
@@ -204,14 +204,16 @@ def del_node(root, data):
     if root is None:
         return root
     if get_height(root.get_right()) - get_height(root.get_left()) == 2:
-        if get_height(root.get_right().get_right()) > \
-           get_height(root.get_right().get_left()):
+        if get_height(root.get_right().get_right()) > get_height(
+            root.get_right().get_left()
+        ):
             root = left_rotation(root)
```
Comment (mateuszz0000) [inline data_structures/binary_tree/avl_tree.py:210]:
```
right_height = get_height(root.get_right().get_right())
right_left_height = get_height(root.get_right().get_left())
if(right_height > right_left_height):
    root = left_rotation(root)
```
Now it's unreadable :cry:

File: data_structures/binary_tree/avl_tree.py [hunk 2]
```diff
@@ -204,14 +204,16 @@ def del_node(root, data):
...
         else:
             root = rl_rotation(root)
     elif get_height(root.get_right()) - get_height(root.get_left()) == -2:
-        if get_height(root.get_left().get_left()) > \
-           get_height(root.get_left().get_right()):
+        if get_height(root.get_left().get_left()) > get_height(
+            root.get_left().get_right()
+        ):
```
Comment (mateuszz0000) [inline data_structures/binary_tree/avl_tree.py:216]:
```
left_height = get_height(root.get_left().get_left())
left_right_height = get_height(root.get_left().get_right())
if(left_height > left_right_height):
    root = right_rotation(root)
```

File: data_structures/binary_tree/avl_tree.py (hunk 1 above)
Comment (cclauss) [reply data_structures/binary_tree/avl_tree.py:210]: The first variable would be `right_right_height` which is no more readable than `get_height(root.get_right().get_right())`.

**Comment by cclauss** [comment]
@mateuszz0000 Your review please

File: data_structures/binary_tree/avl_tree.py (hunk 1 above)
Comment (mateuszz0000) [reply data_structures/binary_tree/avl_tree.py:210]: @cclauss Could we refactor this in this pr? If not i will approve it without and open new with refactor.

**Pull request merged**